smithy-swift-codegen/src/test/kotlin/EnumGeneratorTests.kt

```diff
@@ -39,7 +39,7 @@ class EnumGeneratorTests {
 
         val expectedGeneratedEnum = """
```

Comment:
Below changes are Kotlin codegen tests.  Changes in test expectations:
- All generated structs, unions, enums, & intEnums are Sendable.
- Whitespace around type declarations is made uniform, with an empty line before a declaration & none after.
- Test strings are left-justified to allow for easy maintenance.

Reply:
Try turning off whitespace changes on the Github diff to see the more substantial part of the test changes.